### DISCUSSION TITLE
Feature Request: Reword the rules regarding DMs

### DISCUSSION THREAD

**Discussion by notginned** [TheOdinProject/theodinproject#4766] [General]
### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] The title of this issue follows the `Feature Request: brief description of feature request` format, e.g. `Feature Request: Add a dark mode to the website`
- [X] Would you like to work on this issue?

### Description of the Feature Request

The present verbiage of the rules can be interpreted to suggest that helping people in DMs is okay as long as both parties consent to it.

https://github.com/TheOdinProject/theodinproject/blob/c527256654b4a445999996a265cfc9762db464aa/app/views/guides/community/rules.html.erb#L80

I believe it would be better if we were to explicitly discourage helping people in DMs, with or without their consent for the following reasons:

- Others would be benefited from the conversation as already mentioned
- It would prevent people from giving incorrect, or potentially malicious advice privately, outside of public sight.

### Acceptance criteria

- [ ] The "Do not..." subsection in `public-questions` be reworded to be explicitly discourage helping in DMs

**Comment by JoshDevHub** [comment]
Thank you for making this issue @gingkapls 

I'll see what the team thinks about this. Do you have a specific idea for what you'd want to change the "Do not ..." section of that rule to?

**Comment by notginned** [comment]
One possible phrasing that comes to my mind is:
> Do not use direct messages to help others and do not send direct messages, friend requests, or ping another user without prior consent. 

Though, it seems a bit repetitive. I think the team can express it better than I did, if they agree with my sentiment.

**Comment by neverhack12345** [comment]
I think the "do not" section point on "Send direct messages, friend requests, or ping another user without prior consent" already covers this scenario of helping via DMs as the help itself is in the form of a DM which is a "do not".

The "do" section point on "Help others by guiding them to the solution in a public 1:1 conversation" already covers this scenario explicitly.

**Comment by rlmoser99** [comment]
I am not in favor of making this change, because I do not think our volunteer moderator team needs to more things to moderate. There is no way for us to track when people DM, so we would have to moderate when we see a person asking for permission to DM. This feels like over-moderation to me.

Instead, I think it would be better to add a bot command that when anyone sees a person asking to DM another person, they could step in and make sure they they understand our server expectations. Maybe something like this:

> I see you wanting to connect with another user in DMs. Just as a heads up, in this server we highly recommend asking all questions in public channels. This ensures that you are getting the correct information and others can learn from reading the conversation. In addition, it is unfair to expect specific people to answer your question. If you have received consent, you may DM other people, but you should keep those conversations off-topic and not related to our curriculum.
